src/SubspaceClustering.jl

```diff
@@ -2,4 +2,75 @@ module SubspaceClustering
 
 # Write your package code here.
 
+## K-Subspaces SubspaceClustering
+
+#Imports
+import LinearAlgebra: norm, svd, transpose
+import Base: copy, deepcopy, findall, view
+import Base:argmax
+using Random: randn
+using Logging: @info
+using ProgressLogging: @progress
+using ArnoldiMethod: partialschur
+
+#Exports
+export polar, KSS
+
+#Polar function
+
+function polar(X)
+	U, _, V = svd(X)
+	U*V'
+end
+
+"""
+	KSS(X, d; niters=100)
+
+Run K-subspaces on the data matrix `X`
+with subspace dimensions `d[1], ..., d[K]`,
+treating the columns of `X` as the datapoints.
```

Comment:
This docstring is missing the `Uinit` argument. Please add and describe. See here for an example: https://github.com/dahong67/GCPDecompositions.jl/blob/4900abee06c2e16a0bc70958f10ed7715a6707ab/src/GCPDecompositions.jl#L32-L55